### PR TITLE
Enable some flake8 checks in CI

### PR DESCRIPTION
**Pull request by dmage**

### CODE REVIEW

File: util/config/validators/validate_secscan.py [hunk 1]
```diff
@@ -54,5 +54,5 @@ def validate(cls, validator_context):
             message = str(last_exception)
             raise ConfigValidationException("Could not ping security scanner: %s" % message)
         elif not response.get("state"):
-            message = "Invalid indexer state" % (response.status_code, response.text)
+            message = "Invalid indexer state"
```
Comment (dmage) [inline util/config/validators/validate_secscan.py:57]: @kleesc you authored this line, is my change correct?

File: data/appr_model/channel.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ def get_channel_releases(repo, channel, models_ref):
             Channel.repository == repo,
             Channel.name == channel_name,
             Channel.tag_kind == tag_kind_id,
-            Channel.lifetime_end != None,
+            Channel.lifetime_end.is_null(False),
```
Comment (kleesc) [inline data/appr_model/channel.py:22]: Is that a change with made by Flake8?

File: image/docker/schema2/test/test_manifest.py [hunk 3]
```diff
@@ -293,8 +287,8 @@ def test_get_schema1_manifest():
                 {"created": "2018-04-12T18:37:09.284840891Z", "created_by": "bar"},
             ],
         },
-        "sha256:b5b2b2c507a0944348e0303114d8d93aaaa081732b86451d9bce1f432a537bc7",
-        1885,
+        CONFIG_DIGEST,
+        CONFIG_SIZE,
```
Comment (kleesc) [inline image/docker/schema2/test/test_manifest.py:291]: Were these changes inferred by Flake8? Neat if so.

**Review by kleesc — COMMENTED** [review]
Otherwise LGTM

File: data/appr_model/channel.py (hunk 2 above)
Comment (kleesc) [reply data/appr_model/channel.py:22]: Isn't lifetime_end just an integer?

**Comment by kleesc** [comment]
Do we still need to run `black` as part of CI if this is included, or do they intersect in what they do?

File: data/appr_model/channel.py (hunk 2 above)
Comment (dmage) [reply data/appr_model/channel.py:22]: flake8 complained about `!= None`, and @flavianmissi pointed that `.is_null` is the right way to do it.

https://github.com/quay/quay/blob/d9dbbd88dbb9a1da95ef46195bcfb5458db6a7b8/data/database.py#L1733
If that's the right model, lifetime_end can indeed be null.

File: image/docker/schema2/test/test_manifest.py (hunk 3 above)
Comment (dmage) [reply image/docker/schema2/test/test_manifest.py:291]: Almost, flake8 complained about `\/` in the context where it should be just `/` (as the original json has `\/` that is interpreted as `/`). I could incorrectly replace it with `\\/`, but I decided to fix it, so it has changed its size and digest.

File: data/appr_model/channel.py (hunk 2 above)
Comment (kleesc) [reply data/appr_model/channel.py:22]: Does it complain on `is not None`?

Comment (kleesc) [reply data/appr_model/channel.py:22]: Ah, it's peewee's wrapper for  numbers.

Comment (dmage) [reply data/appr_model/channel.py:22]: It doesn't complain on `is not None`, but `is not None` has different semantics and breaks tests.

**Review by kleesc — APPROVED** [review]

**Pull request merged**